modules/platforms/dotnet/Apache.Ignite.Core.Tests/ApiParity/ClusterParityTest.cs

```diff
@@ -36,7 +36,8 @@ public class ClusterParityTest
             "isBaselineAutoAdjustEnabled",
             "baselineAutoAdjustEnabled",
             "baselineAutoAdjustTimeout",
-            "baselineAutoAdjustStatus"
+            "baselineAutoAdjustStatus",
+            "readOnly"
```

Comment:
If readonly is not needed, for now, in .NET, it is probably will be required in the future. Consider creating an issue/discuss this topic at https://lists.apache.org/thread.html/a8a585804578b80545e5d4f862f644857858c68124a1c847b9b2c906@%3Cdev.ignite.apache.org%3E

Reply:
The issue already created https://issues.apache.org/jira/browse/IGNITE-11863